pkg/ingester/ingester.go

```diff
@@ -3154,23 +3154,6 @@ func (i *Ingester) Push(ctx context.Context, req *mimirpb.WriteRequest) (*mimirp
 	return nil, handledErr
 }
 
-func handlePushError(err error) error {
```

Comment:
Note for reviewers: this method has been moved from `ingester.go` to `errors.go`.